app/validations/password.js

```diff
@@ -1,12 +1,12 @@
 import {
   validatePresence,
-  validateLength,
+  validateFormat,
 } from 'ember-changeset-validations/validators';
 import messages from './custom-messages';
 
 export default {
   password: [
-    validateLength({ min: 8, allowBlank: true, message: messages.passwordRules }),
+    validateFormat({regex: /^(?=.*?[0-9]).{8,}$/, allowBlank: true, message: messages.passwordRules }),
```

Comment:
but. this won't match for a password with a blank space in the middle. wht about `^(?=[\S]*?[0-9]).{8,}$`?